maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -38,6 +38,10 @@ public class CLIManager {
 
     public static final char BATCH_MODE = 'B';
 
+    public static final String NON_INTERACTIVE = "ni";
+
+    public static final String FORCE_INTERACTIVE = "fi";
```

Comment:
I wonder whether such important flags should really have short options, especially `force`. Keeping `-B` as short hand is fine.